chapter2-partial.md

```diff
@@ -139,6 +139,21 @@ Kopyalanan dosya aynı klasör içine değil, farklı bir klasör içine kopyala
 
 %%%% TODO: cp komutu örneği geliştirilmeli, PATH (relative and absolute) kavramı bahsedilmeli
 
+Path işletim sistemimizin komut satırından veya terminal penceresinden gerekli çalıştırılabilir dosyaların yerini belirlemek için kullandığı sistem değişkeni veya yol olarak tanımlanır.
```

Comment:
"sistem değişkeni" ifadesi ile belirtilen $PATH ile cp komutu için bahsettiğimiz path arasında fark var..